### PR TITLE
se redacto manual de sistema

### PR DESCRIPTION
**Pull request by catalinapinillas**
## Describe the PR content and subject

Se redactó manual de Sistemas abarcando factoring (facturas y documentos) y dejando de manera inconclusa Quazar, se afinaron detalles de redacción, títulos, encabezados y formatos de imágen.
## Proof of completion
 
image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Not bug related code refactor

## Checklisk
 
- [x]  Ensure the code follows the project's coding standards
    - [x]  Choose descriptive and unambiguous names.
    - [x]  Make meaningful distinctions.
    - [x]  Use pronounceable names.
    - [x]  Use searchable names.
    - [x]  Replace magic numbers with named constants. (Also magic text)
    - [x]  Avoid encodings. Don’t append prefixes or type information.
- [x]  Check if the new code has proper unit tests
- [x]  Verify that the code is properly commented for understandability
    - [x]  Remove poor quality, not meaningful comments
    - [x]  Use comments on non standard code and hard
- [x]  Check if the code changes are relevant to the pull request description
- [x]  Ensure the new code does not introduce any new bugs
- [x]  Check if the pull request contains minimal and logical commits
- [x]  Check if the code changes affect performance and suggest improvements if it does
- [ ]  Review if the pull request resolves the issue it is linked to
    - [x]  If not, describe why the PR y needed/split.  Quedaró pendiente la redacción de Clientes y matriz de riesgo, además de correcciones de formato, tipeo y terminar la sección de Quazar.
- [x]  Do not include extensive refactoring if not requested
- [x]  Ensure all continuous integration checks are passed
- [x]  Provide proof of completion on issue or pull request

### CODE REVIEW

File: _posts/2024-06-04-factoring-documentos.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+</p>
+
+En la última columna encontrarás el estado de *cobranza* que en caso que el documento cuente con información en ese apartado, podrás consultar aquí la información.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/cobranzadoc1.png" alt="cobranzadoc1">
+</p>
+
+
+*Aquí puedes ver la columna mencionada*
```
Comment (dquinteros) [inline _posts/2024-06-04-factoring-documentos.md:31]: Colocar la referencia a la imagen por encima de la imagen.

File: _posts/2024-06-05-factoring-documento-vigente.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: ¿Quieres saber si el documento que buscas esta vigente?, aqui te indicamos como.
```
Comment (dquinteros) [inline _posts/2024-06-05-factoring-documento-vigente.md:3]:
```suggestion
title: ¿Quieres saber si el documento que buscas esta vigente?, aquí te indicamos como.
```

File: _posts/2024-06-06-factoring-documentos-mora.md
Comment (dquinteros) [inline _posts/2024-06-06-factoring-documentos-mora.md:1]: Mezclar este pors con el de vigente.

**Review by dquinteros — CHANGES_REQUESTED** [review]

File: _posts/2024-06-02-factoring-facturas-filtros.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Filtra la información en pantalla y  sintetizar los resultados.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-02-factoring-facturas-filtros.md]: listo

File: _posts/2024-06-02-factoring-facturas-filtros.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+comments: false
+author: Catalina Pinilla
+---
+Junto con los filtros de Cliente, Ejecutivo, Rango de fecha y Folio, cuentas con la opción de filtrar por *Deudor*.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/filtrodefiltrofacturas1.png" alt="filtrodefiltrofacturas1">
+</p>
+
+Con esto podrás ver la información solo de ese Deudor, además junto con eso podrás seleccionar las columnas que quieres que se visualicen, eliminando las que no tienen información relevante para el caso que buscas.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-02-factoring-facturas-filtros.md]: listo :)

File: _posts/2024-06-02-factoring-facturas-filtros.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Filtra la información en pantalla y  sintetizar los resultados.
+subtitle: Facturas - Filtros
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [facturas, comercial, burcar, filtro]
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-02-factoring-facturas-filtros.md]: listo :)

File: _posts/2024-06-03-factoring-facturas-rango.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Acitva Rangos de fecha y el filtro CRM para ver que facturas podemos comprar. 
+subtitle: Facturas - Rangos y facturas disponibles.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [facturas, comercial, burcar, filtro, compra]
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-03-factoring-facturas-rango.md]: listo :)

File: _posts/2024-06-05-factoring-documento-vigente.md (hunk 2 above)
Comment (catalinapinillas) [reply _posts/2024-06-05-factoring-documento-vigente.md]: listo :)

File: _posts/2024-06-01-factoring-facturas-seleccion.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+ Podrás ver que aparecerá un gran listado en un orden predeterminado. Te encontraras con el uso de *filtros* los cuales serán: Clientes, Ejecutivos, Rango de fecha y Folio, esto dependiendo de como  necesites enfocar tu búsqueda, mientras más datos ingreses, más acotados serán los resultados reflejados.
+ 
+ Si necesitas eliminar tus selecciones puedes limpiar los filtros que aplicaste en *Limpiar filtros*.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/filtrofactura3.png" alt="limpiarfiltro">
+</p>
+
+ *aquí un ejemplo*
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-01-factoring-facturas-seleccion.md:20]: Aún no resuelto

File: _posts/2024-06-02-factoring-facturas-filtros.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  <img src="https://cdn.capitalexpress.cl/img/filtrodefiltrofacturas1.png" alt="filtrodefiltrofacturas1">
+</p>
+
+Con esto podrás ver la información solo de ese Deudor, además junto con eso podrás seleccionar las columnas que quieres que se visualicen, eliminando las que no tienen información relevante para el caso que buscas. 
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/filtrodefiltrofacturas3.png" alt="filtrodefiltrofacturas3">
+</p>
+
+Con  los filtros que necesitas consultar activos podrás ver la información precisa que estas buscando sin datos que te interrumpan en tu negocio.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-02-factoring-facturas-filtros.md]: Listo :)

File: _posts/2024-06-02-factoring-facturas-filtros.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+layout: post
+title: Filtra la información en pantalla y  sintetizar los resultados.
+subtitle: Facturas - Filtros
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [facturas, comercial, burcar, filtro]
+comments: false
+author: Catalina Pinilla
+---
+Junto con los filtros de Cliente, Ejecutivo, Rango de fecha y Folio, cuentas con la opción de filtrar por *Deudor*.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-02-factoring-facturas-filtros.md:11]: Cambio no aplicado

File: _posts/2024-06-03-factoring-facturas-rango.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Acitva Rangos de fecha y el filtro CRM para ver que facturas podemos comprar.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-03-factoring-facturas-rango.md:3]: Cambio no aplicado

File: _posts/2024-06-03-factoring-facturas-rango.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Acitva Rangos de fecha y el filtro CRM para ver que facturas podemos comprar. 
+subtitle: Facturas - Rangos y facturas disponibles.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [facturas, comercial, buscar, filtro, ventas, compras búsqueda]
```
Comment (dquinteros) [inline _posts/2024-06-03-factoring-facturas-rango.md:7]:
```suggestion
tags: [facturas, comercial, buscar, filtro, ventas, compras, búsqueda]
```

File: _posts/2024-06-03-factoring-facturas-rango.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+author: Catalina Pinilla
+---
+En pantalla podrás ver el recuadro de *Seleccionar fecha*.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/fecha.png" alt="fecha">
+</p>
+
+
+Podrás notar que el listado se filtró, junto con esto debes activar el switch de Crm para que asi veas reflejadas solo las facturas que pueden ser compradas por **Capital Express Servicios Financieros** en tu pantalla.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-03-factoring-facturas-rango.md:18]: Cambio no aplicado

File: _posts/2024-06-04-factoring-documentos.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Quieres ver los documentos que han sido adquiridos por Capital Express Servicios Financieros, ingresa de esta forma.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-04-factoring-documentos.md:3]: Cambio no aplicado

File: _posts/2024-06-04-factoring-documentos.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+subtitle: Factoring - Documentos.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [documentos, comercial, cobranza]
+comments: false
+author: Catalina Pinilla
+---
+Para navegar en este apartado contaras con los filtros de Cliente, Ejecutivo, Rango de fecha y folio. Nos encontraremos con todos los Documentos que han sido adquiridos por **Capital Express Servicios Financieros**, Facturas, cheques, Créditos directos, etc. Contarás con la opción de *Limpiar Filtros* en caso que quieras reiniciar tu busqueda.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-04-factoring-documentos.md:11]: Cambio no aplicado

File: _posts/2024-06-04-factoring-documentos.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+author: Catalina Pinilla
+---
+Para navegar en este apartado contaras con los filtros de Cliente, Ejecutivo, Rango de fecha y folio. Nos encontraremos con todos los Documentos que han sido adquiridos por **Capital Express Servicios Financieros**, Facturas, cheques, Créditos directos, etc. Contarás con la opción de *Limpiar Filtros* en caso que quieras reiniciar tu busqueda. 
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/docfiltro1.png" alt="docfiltro1">
+</p>
+
+
+También contaremos con un filtrado de estado, que ayudará a ver si el documento se encuentra Vigente, Moroso, Cancelado, etc. Para tener una visión de como proceder con cada caso.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-04-factoring-documentos.md:18]: cambio no aplicado

File: _posts/2024-06-05-factoring-documento-vigente.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+subtitle: Documentos - Estado vigente.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [documentos, comercial, vigente, buscar, filtro]
+comments: false
+author: Catalina Pinilla
+---
+
+Según el Cliente al cual quieres buscar, es aconsejable que indiques un periodo de fechas en el cual estas realizando la consulta, aplica la mayor cantidad de filtros en la parte superior para que la información siempre sea lo más acotada posible. Aquí nos mostrarán todos los documentos comprados por **Capital Express Servicios Financieros**, pero para saber cuales tienen un estado *vigente* utilizaremos la herramienta *filtrar por estado* y ahí dejaremos seleccionada solo la opción vigente y obtendremos lo que buscamos.
```
[the remark this answers is not in the document]
Comment (dquinteros) [reply _posts/2024-06-05-factoring-documento-vigente.md:12]: Cambio no aplicado

File: _posts/2024-06-07-factoring-quazar-cyv-fitros.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+
+Otro filtro muy útil es el de *Calendario* el cual puedes usar si necesitas consultar un periodo en específico o acotar un rango de fechas que tú necesites definir, lo encontraras en *Selecciona una fecha*
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/calendarioquazar.png" alt="calendarioquazar">
+</p>
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/calendarioquazar2.png" alt="calendarioquazar2">
+</p>
```
Comment (dquinteros) [inline _posts/2024-06-07-factoring-quazar-cyv-fitros.md:34]: Quiza es mejor colocar esta imagenes lado a lado en luega de una encima de la otra

File: _posts/2024-06-08-factoring-quazar-competencia.md [hunk 18]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Necesitas ver como trabaja tu Cliete con los otros Factoring, te indicaremos como.
```
Comment (dquinteros) [inline _posts/2024-06-08-factoring-quazar-competencia.md:3]:
```suggestion
title: ¿Quieres evaluar tu competencia¿, te indicaremos como.
```

File: _posts/2024-06-08-factoring-quazar-competencia.md [hunk 19]
```diff
@@ -0,0 +1,18 @@
...
+subtitle: Clientes- Clientes
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [quazar, comercial, riesgo, factoring, filtros, competencia]
+comments: false
+author: Catalina Pinilla
+---
+Se mostrán dos tablas donde verás el detalle de como el Cliente ha operado con los distintos Factoring a los que ha acudido, en la superior verás el porcentaje y los montos de facturas que cada Factorinfg  ha adquirido al Cliente y el desglose de el total de facturas cedidas por mes.
```
Comment (dquinteros) [inline _posts/2024-06-08-factoring-quazar-competencia.md:11]:
```suggestion
Se muestran dos tablas donde verás el detalle de las opreaciones de tus clientes con los distintos Factoring a los que ha acudido, en la superior verás el porcentaje y los montos de facturas que cada Factoring ha adquirido al Cliente y el desglose del total de facturas cedidas por mes.
```

**Review by dquinteros — CHANGES_REQUESTED** [review]

File: _posts/2024-06-06-quasar-comprayventa.md [hunk 20]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Ve los detalles de compra y venta de los documentos según tus Clientes, de manera organizada y eficiente.
+subtitle: Quazar Compras, ventas y Factorizacion.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-06-quasar-comprayventa.md]: Listo :)

File: _posts/2024-06-02-factoring-facturas-filtros.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-02-factoring-facturas-filtros.md]: creo que ahora fue aplicado

File: _posts/2024-06-03-factoring-facturas-rango.md (hunk 11 above)
Comment (catalinapinillas) [reply _posts/2024-06-03-factoring-facturas-rango.md]: listo

File: _posts/2024-06-04-factoring-documentos.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-04-factoring-documentos.md]: listo

File: _posts/2024-06-03-factoring-facturas-rango.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-03-factoring-facturas-rango.md]: listo :)

File: _posts/2024-06-04-factoring-documentos.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-04-factoring-documentos.md]: listo

File: _posts/2024-06-05-factoring-documento-vigente.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-05-factoring-documento-vigente.md]: listo :)

File: _posts/2024-06-07-factoring-quazar-cyv-fitros.md [hunk 21]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Sintetiza la información de los documentos de tu Cliente
+subtitle: Quazar - Compras, ventas y Factorizacion filtros
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-07-factoring-quazar-cyv-fitros.md]: listo :)

File: _posts/2024-06-08-factoring-quazar-competencia.md (hunk 18 above)
Comment (catalinapinillas) [reply _posts/2024-06-08-factoring-quazar-competencia.md]: listo

File: _posts/2024-06-08-factoring-quazar-competencia.md (hunk 19 above)
Comment (catalinapinillas) [reply _posts/2024-06-08-factoring-quazar-competencia.md]: listo

File: _posts/2024-06-06-quasar-comprayventa.md [hunk 22]
```diff
@@ -0,0 +1,23 @@
...
+---
+Para comenzar debes seleccionar los filtros básicos para poder realizar una consulta, Cliente y Rango de fecha para que comience a arrojar resultados, sin esto no obtendremos información en la pantalla.
+
+Al lado derecho, podemos ver las empresas de Factoring ordenados en preferencia del Cliente en relación a Documentos cedidos, apareciendo en primer lugar el que lidera los negocios y el total que aparece ahi, corresponde a los documentos Cedidos, más los Cedidos Reclamados.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/topfactoring2.png" alt="topfactoring">
+</p>
+
+En la parte inferior veras el detalle de la tabla de superior, pero separada por periodos, estos estaran separados por rangos de los días del mes y los que a su vez se encontrarán separados por los meses del año en curso.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-06-quasar-comprayventa.md]: listo

File: _posts/2024-06-05-factoring-documento-vigente.md [hunk 23]
```diff
@@ -0,0 +1,32 @@
...
+  <img src="https://cdn.capitalexpress.cl/img/filtromoradoc2.png" alt="filtroporestadodoc2">
+</p>
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/estadovigentedoc.png" alt="estadovigentedoc">
+</p>
+
+Haciendo estos pasos podrás ver los documentos vigentes que tiene el Cliente seleccionado en estos momentos y en el periodo de tiempo seleccionado. 
+
+Ten los datos básicos para iniciar una busqueda como nombre del Cliente y rango de fecha, recuerda que meintras más datos ingreses más acotados serán los resultados. Seguido de eso en *Filtrar por estado* selleciona la opción *Moroso* y verás como ahora solo te muestran los documentos con estado de mora.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-05-factoring-documento-vigente.md]: listo

File: _posts/2024-06-07-factoring-quazar-cyv-fitros.md (hunk 17 above)
Comment (catalinapinillas) [reply _posts/2024-06-07-factoring-quazar-cyv-fitros.md]: listo, gracias por tu ayuda 
PATITTA DE GATO

File: _posts/2024-06-05-factoring-documento-vigente.md [hunk 24]
```diff
@@ -0,0 +1,22 @@
...
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/filtromoradoc2.png" alt="filtroporestadodoc2">
+</p>
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/estadovigentedoc.png" alt="estadovigentedoc">
+</p>
+
+Haciendo estos pasos podrás ver los documentos vigentes que tiene el Cliente seleccionado en estos momentos y en el periodo de tiempo seleccionado.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-05-factoring-documento-vigente.md]: listo

File: _posts/2024-06-01-factoring-facturas-seleccion.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-01-factoring-facturas-seleccion.md:17]: listo

File: _posts/2024-06-06-quasar-comprayventa.md [hunk 25]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Ve los detalles de compra y venta de los documentos según tus Clientes, de manera organizada y eficiente.
+subtitle: Quazar Compras, ventas y Factorización. 
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [quazar, comercial, riesgo, factoring]
```
Comment (dquinteros) [inline _posts/2024-06-06-quasar-comprayventa.md:7]:
```suggestion
tags: [comercial, riesgo, factoring]
```

File: _posts/2024-06-18-factoring-quazar-cyv-fitros.md
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-18-factoring-quazar-cyv-fitros.md:1]: listo

File: _posts/2024-06-18-quasar-comprayventa.md [hunk 26]
```diff
@@ -0,0 +1,23 @@
...
+title: Ve los detalles de compra y venta de los documentos según tus Clientes, de manera organizada y eficiente.
+
+subtitle: Quazar Compras, ventas y Factorizacion. 
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [quazar, comercial, riesgo, factoring]
+comments: false
+author: Catalina Pinilla
+---
+## Aquí podrás ver los detalles de compra y venta de los documentos según tus Clientes.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-18-quasar-comprayventa.md]: listo

File: _posts/2024-06-18-quasar-comprayventa.md [hunk 27]
```diff
@@ -0,0 +1,23 @@
...
+
+Para comenzar debes seleccionar los filtros básicos para poder realizar una consulta, Cliente y Rango de fecha para que comience a arrojar resultados, sin esto no obtendremos información en la pantalla.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-18-quasar-comprayventa.md]: listo

File: _posts/2024-06-07-factoring-quazar-cyv-fitros.md [hunk 28]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Sintetiza la información de los documentos de tu Cliente
+subtitle: Compras, ventas y Factorización
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [quazar, comercial, riesgo, factoring, filtros]
```
Comment (dquinteros) [inline _posts/2024-06-07-factoring-quazar-cyv-fitros.md:7]:
```suggestion
tags: [comercial, riesgo, factoring, filtros]
```

Comment (catalinapinillas) [reply _posts/2024-06-07-factoring-quazar-cyv-fitros.md]: listo

File: _posts/2024-06-06-quasar-comprayventa.md (hunk 25 above)
Comment (catalinapinillas) [reply _posts/2024-06-06-quasar-comprayventa.md]: listo

**Review by dquinteros — APPROVED** [review]

File: _posts/2024-06-03-factoring-facturas-rango.md [hunk 29]
```diff
@@ -0,0 +1,24 @@
...
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/crmboton1.png" alt="crmboton">
+</p>
+
+Así estarás seguro que los docuemntos en el listado son los que se pueden adquirir.
```
[the remark this answers is not in the document]
Comment (catalinapinillas) [reply _posts/2024-06-03-factoring-facturas-rango.md]: listo

File: _posts/2024-06-04-factoring-documentos.md (hunk 1 above)
Comment (catalinapinillas) [reply _posts/2024-06-04-factoring-documentos.md]: listo

File: _posts/2024-06-06-factoring-documentos-mora.md
Comment (catalinapinillas) [reply _posts/2024-06-06-factoring-documentos-mora.md:1]: listo

**Pull request merged**